### PR TITLE
CI  Issue fixed and images optimised #413   resolved to #412

### PR DESCRIPTION
**Pull request by mukeshblackhat**
Created a PR WRT to https://github.com/tailcallhq/tailcallhq.github.io/pull/412

Proof read identify gaps
Make the build pass
Optimize uploaded images
#413 resolved
/claim #413

### CODE REVIEW

**Comment by tusharmath** [comment]
Can you fix the conflicts?

**Comment by mukeshblackhat** [comment]
these two conflicts are with the images , just changing them and even in the file also i made some liitle changes you can accept the incoming changes and it will work , i dont have write access to resolve the conflicts

**Comment by mukeshblackhat** [comment]
> Can you fix the conflicts?

merge conflicts resolved

File: blog/tailcall-n+1-working-2024-08-04.md [hunk 1]
```diff
@@ -21,20 +21,20 @@ Unlike a traditional GraphQL implementation where the resolvers are written by h
...
 To see the actual implementation you can check out the [tracker.rs](https://github.com/tailcallhq/tailcall/blob/main/src/core/config/npo/tracker.rs) implementation.
 :::
 
 We essentially use a Depth-First Search (DFS) algorithm starting at the root query and traversing all the connected nodes. The algorithm works as follows:
 
-1. Initialize a to variables to track the currently traversed path and visited fields so that we can avoid cycles.
+1. Initialize variables to track the currently traversed path and visited fields to avoid cycles.
 2. Start at the root query and begin traversing the graph data structure.
-3. For each field in the current node, check if it has a resolver and is not batched. We know if the node contains a resolver if that node has a [`@http`](../docs/directives.md#http-directive) or a [`@grpc`](../docs/directives.md#grpc-directive).
+3. For each field in the current node, check if it has a resolver and is not batched. We know if the node contains a resolver if that node has a [`@http`](https://tailcall.run/docs/directives.md#http-directive) or a [`@grpc`](https://tailcall.run/docs/directives.md#grpc-directive).
```
Comment (tusharmath) [inline blog/tailcall-n+1-working-2024-08-04.md:34]:
```suggestion
3. For each field in the current node, check if it has a resolver and is not batched. We know if the node contains a resolver if that node has a [`@http`](/docs/directives.md#http-directive) or a [`@grpc`](/docs/directives.md#grpc-directive).
```

Dont' use absolute URLs.

Comment (mukeshblackhat) [reply blog/tailcall-n+1-working-2024-08-04.md:34]: how you are setting the configurations for this conversion 

[WARNING] Blog markdown link couldn't be resolved: (../docs/N+1.md) in source file "/Users/mukeshbishnoi/Desktop/open source/tailcallhq.github.io/blog/tailcall-n+1-working-2024-08-04.md"

**Comment by tusharmath** [comment]
Moving to draft to reduce noise and improve CI efficiency. Once you are ready just mark it as "ready to review". Feel free to give a shoutout on the #contributors channel on Discord if you want immediate attention.

**Comment by tusharmath** [comment]
Sorry @mukeshblackhat had to close this PR. We are in a bit of a hurry to deploy this blog. Thanks once again for your contributions!

**Comment by mukeshblackhat** [comment]
@tusharmath why redirection is not working properly in this file ??

**Pull request closed without merging**